man/org.freedesktop.systemd1.xml

```diff
@@ -340,6 +340,10 @@ node /org/freedesktop/systemd1 {
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
       readonly t FinishTimestampMonotonic = ...;
       @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t SoftRebootStartTimestamp = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
+      readonly t SoftRebootStartTimestampMonotonic = ...;
+      @org.freedesktop.DBus.Property.EmitsChangedSignal("const")
```

Comment:
hmm, why make this about soft reboot only? I think it would make more sense to maintain a timestamps for the beginning of any form of shutdown. i.e. if we add this then i'd focus on the bigger picture, and I am pretty sure that there could be code that runs during shutdown might want to know when shutdown was initiated.

Reply:
Because it's going to be lost in other cases, and it's too late to be useful for anything in particular

Reply:
hmm? why would it be lost? while the shutdown is in progress there's so much time to still do something with the data after all pretty much all code is still running initiially during shutdown and could query this data from pid 1.

please make this generic